test/udp_sender.js

```diff
@@ -95,51 +95,63 @@ describe('udp sender should', () => {
         sender.flush();
     });
 
-    it ('span references serialize', (done) => {
+    describe('span reference tests', () => {
+        let tracer = new Tracer(
+            'test-service-name',
+            new InMemoryReporter(),
+            new ConstSampler(true)
+        );
         let parentContext = tracer.startSpan('just-used-for-context').context();
-        let childOfRef = new opentracing.Reference(opentracing.REFERENCE_CHILD_OF, tracer.startSpan('just-used-for-context').context());
-        let followsFromRef = new opentracing.Reference(tracer.startSpan('just-used-for-context').context());
+        let childOfContext = tracer.startSpan('just-used-for-context').context();
```

Comment:
I don't see why you need a new span for childOfContext instead of just using `parentContext`, but whatever